agg-ast/ast/src/definitions.rs

```diff
@@ -1158,16 +1162,12 @@ pub enum TaggedOperator {
     LTrim(Trim),
     #[serde(rename = "$rtrim")]
     RTrim(Trim),
-
-    // Subquery Operators (extended from MQL)
-    #[serde(rename = "$regexFind")]
-    RegexFind(RegexFind),
-    #[serde(rename = "$regexFindAll")]
-    RegexFindAll(RegexFindAll),
     #[serde(rename = "$replaceAll")]
```

Comment:
There was no reason to use different structs here since they have the same fields. Makes the code cleaner